check_log_ng.py

```diff
@@ -613,6 +613,10 @@ def _check_log(self, logfile, seekfile):
         _debug("logfile='{0}', seekfile='{1}'".format(logfile, seekfile))
         logfile = LogChecker.to_unicode(logfile)
         if not os.path.exists(logfile):
+            # if set error_no_exist, return UNKNOWN.
+            if self.config['error_no_exist']:
+                self.message = "UNKNOWN: Logfile is not found."
```

Comment:
I think it's better to contain logfile path in output.

Reply:
OK. I modified to display logfile path.